internal/bminventory/inventory.go

```diff
@@ -1576,6 +1590,25 @@ func (b *bareMetalInventory) UpdateClusterInstallConfigInternal(ctx context.Cont
 	return &cluster, nil
 }
 
+func (b *bareMetalInventory) setInstallConfigOverridesUsage(usages usage.FeatureUsage, installConfigOverrides map[string]interface{}, clusterID strfmt.UUID) {
+	props := usages[usage.InstallConfigOverrides].Data
+	if props == nil {
+		props = make(map[string]interface{})
+	}
+	for k, v := range installConfigOverrides {
+		key := k
+		switch v := v.(type) {
+		case map[string]interface{}:
+			for i := range v {
+				key = key + " " + i
+			}
+		}
+		props[key] = true
+	}
```

Comment:
What is this all doing?

Reply:
It transforms the install config overrides into the feature usage map 
So going from this
```
'"{\"networking\":{\"networkType\": \"OVNKubernetes\"},\"fips\":true}"'
```
to this
```
       "fips": true,
       "networking networkType": true

```